ipatests/test_integration/test_installation.py

```diff
@@ -1208,3 +1208,36 @@ def test_replica_install_against_server(self):
                                             self.replicas[0].hostname],
                                            stdin_text=dirman_password)
         assert self.replicas[0].hostname not in cmd.stdout_text
+
+
+class TestInstallMasterClient(IntegrationTest):
```

Comment:
This scenario is not related with installation, so IMO should not be part of test_installation.py. We can add a new test file test_cert.py which can contain cert related tests.

Reply:
done